kytos/core/switch.py

```diff
@@ -191,6 +192,24 @@ def as_json(self):
         return json.dumps(self.as_dict())
 
 
+class Connection_states(Enum):
```

Comment:
I agree with the caps, I wrote with caps first but the changes it since it was a class...
I'll capitalize it back